src/task.rs

```diff
@@ -1220,6 +1233,13 @@ pub struct TaskMetrics {
     /// ```
     pub total_slow_poll_count: u64,
 
+    /// The total count of tasks with long scheduling delays.
+    ///
+    /// This is defined as tasks taking longer than
+    /// [`long_delay_threshold`][TaskMonitor::long_delay_threshold] to be executed aftering being
+    /// scheduled.
```

Comment:
nit: to be very pedantic, long is defined as equal to or greater than `long_delay_threshold`, so perhaps this could be:

```suggestion
    /// This is defined as tasks taking
    /// [`long_delay_threshold`][TaskMonitor::long_delay_threshold] or longer to be executed
    /// after being scheduled.
```

(also, there was a typo: "aftering")